### ISSUE TITLE
hydra-client-dotnet scope regex validation fails

### ISSUE THREAD

**Issue by daniel-eichinger-bl** [ory/hydra#3142]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/hydra/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/hydra/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

The regex used in the [hydra-client-dotnet](https://github.com/ory/hydra-client-dotnet) scope's validation `([a-zA-Z0-9\\.\\*]+\\s?)+` ([see HydraOAuth2Client.cs:72](https://github.com/ory/hydra-client-dotnet/blob/bad06c83fd9b12d5c3dce23e59fe18822d314744/src/Ory.Hydra.Client/Model/HydraOAuth2Client.cs#L772)) does not match any scopes e.g., `openid`.

But the scope regex comment to generate the OpenAPI specification is correct `([a-zA-Z0-9\.\*]+\s?)+` [client/client.go:72](https://github.com/ory/hydra/blob/master/client/client.go#L72).

If the two regexes are compared the one in the .NET SDK has an additional slash: 
- `([a-zA-Z0-9\\.\\*]+\\s?)+` 
- `([a-zA-Z0-9\.\*]+\s?)+`

## w/ .NET 6.0 Console App
```
using System.Text.RegularExpressions;

Regex regexScope = new Regex(@"([a-zA-Z0-9\\.\\*]+\\s?)+", RegexOptions.CultureInvariant);
Console.WriteLine(regexScope.Match("openid").Success); 
```
## w/ hydra-client-dotnet
1. have Ory Hydra running w/ `dynamic_client_registration` enabled 
2. have a .NET WebAPI project that uses the [nuget Ory Hydra Client](https://www.nuget.org/packages/Ory.Hydra.Client/)
- WebAPI project has a Post endpoint, where a `HydraOAuth2Client` can be sent to
- .NET will call the Validate method of the `OryHydraOAuth2Client` automatically, because when a model class implements IValidatableObject interface, the Model Binder automatically identifies it and calls Validate of that class during model binding
```
[HttpPost]
public IActionResult Post(HydraOAuth2Client client){  
   return Ok();
}
```
3. make a post request e.g.,
```
{
    "client_id": "test-client",
    "scope" : "openid"
}
```
4. hydra dotnet sdk will give a 400, because Regex does could not match scope
```
{
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
    "title": "One or more validation errors occurred.",
    "status": 400,
    "traceId": "00-75f0f3e04467849f176e1767262880b2-3455b982ad41c58e-00",
    "errors": {
        "Scope": [
            "Invalid value for Scope, must match a pattern of ([a-zA-Z0-9\\\\.\\\\*]+\\\\s?)+"
        ]
    }
}
```

### Version

1.11.8

### On which operating system are you observing this issue?

Windows

### In which environment are you deploying?

Binary

**Comment by aeneasr** [comment]
Probably introduced by: https://github.com/OpenAPITools/openapi-generator/pull/2948

This will need to be fixed in OpenAPI Generator - alternatively we can also remove the pattern from the OpenAPI spec in Ory Hydra which is probably a good idea anyways.

**Comment by aeneasr** [comment]
Will be fixed in 2.0
